client/my-sites/theme/main.jsx

```diff
@@ -760,6 +782,20 @@ class ThemeSheet extends Component {
 		);
 	};
 
+	onThemeActive = () => {
```

Comment:
Nit: Maybe call this `onAtomicThemeActive()` to make it clear that the function is intended for Atomic sites?